lib/taurus/qt/qtgui/taurusgui/config_loader/sardana.py

```diff
@@ -0,0 +1,206 @@
+#############################################################################
+##
+# This file is part of Taurus
+##
+# http://taurus-scada.org
+##
+# Copyright 2011 CELLS / ALBA Synchrotron, Bellaterra, Spain
+##
+# Taurus is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+##
+# Taurus is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+##
+# You should have received a copy of the GNU Lesser General Public License
+# along with Taurus.  If not, see <http://www.gnu.org/licenses/>.
+##
+###########################################################################
+
+""""""
+
+import sys
+
+import taurus
+from taurus.external.qt import Qt
+from taurus.qt.qtgui.taurusgui.config_loader.abstract import \
+    AbstractConfigLoader
+from taurus.qt.qtgui.taurusgui.utils import PanelDescription
+
+__all__ = ["SardanaConfigLoader"]
+
+
+class SardanaConfigLoader(AbstractConfigLoader):
```

Comment:
Ideally, this config loader should be moved to the sardana project, and the sardana module would register the plugin.
Unfortunately, this makes things a bit more complex regarding backwards-compatibility (specially because if we want to support py2 we would need to backport it to the sardana 2.x branch, or create a separate taurus-legacy-plugin project).
Apart, it also would make it a bit more difficult to test and spot regressions while developing this PR.
So I propose to keep it here until the PR is ready and tested and then consider if we can move it out of taurus before merging

Reply:
I think that, too.